tests/unittest/llmapi/test_model_feature_fallbacks.py

```
@@ -0,0 +1,387 @@
+        assert mock.args.cuda_graph_config is cuda_cfg
+
+    def test_cuda_graph_disabled_when_unsupported(self):
+        """CUDA graph should be disabled when model has NO status (mocked)."""
+        cuda_cfg = SimpleNamespace(mode="static")
+        mock = _make_mock_llm(
+            architecture="MockUnsupportedModel",
+            cuda_graph_config=cuda_cfg,
+        )
+
+        # Mock get_support_status to return NO for CUDA_GRAPH
+        def mock_status(arch, feature):
```

Ruff (0.14.11)

Unused function argument: `arch`

(ARG001)